concourse/model/traits/release.py

```diff
@@ -67,10 +85,11 @@ class ReleaseNotesPolicy(enum.Enum):
     ),
     AttributeSpec.optional(
         name='release_notes_policy',
-        default=ReleaseNotesPolicy.DEFAULT,
+        default=ReleaseNotesPolicy.DEFAULT.value,
```

Comment:
nit: would it not be better to use the "real" value as default value (the enum instance) and do the conversion in the getter exposing the attr? this holds also true for other occurrences..